rest/http_api.md

```diff
@@ -1288,3 +1288,114 @@ Content-Type: application/json
  ]
 }
 ```
+## Locations ##
+
+### List locations
+
+List all the existent location definitions. 'Content-Type' header should be set to 'application/json'.
+
+`GET    /locations`
+
+**Response**:
+
+```HTTP
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+  "locations":[
+    {
+      "name":"location1",
```

Comment:
Actually this call doesn't provide type and properties details for each location like shown in this example.
It provides a list of references, like:
```json
{"locations":[{"rel":"location","href":"/locations/myLocation1","type":"application/json"},{"rel":"location","href":"/locations/myLocation2","type":"application/json"}]}
```

And another endpoint should be documented to get a location description:
GET /locations/<locationName>